### PR TITLE
Attack cost tweaks to a select few monsters

### PR DESCRIPTION
**Pull request by SenpaiSlime**
#### Summary
Balance "Horse, reindeer and zeindeer attack cost balanced"

#### Purpose of change
A select few enemies, upon a second once over, still fit the bill for an adjustment to their attack cost. Namely the zeindeer, reindeer and horse. The three of these creatures all have a very high movement speed, and likely are not intentionally meant to attack as fast as they do. With attack speed defaulting to 100 without being defined, these enemies are much more dangerous than what would be sensible, attacking harder or cumulatively harder than more sensible threats, i.e. horses are much more capable of destroying you in a short period of time than a moose.

#### Describe the solution
I have gone and added attack cost values to better throttle the damage potential of these monsters, the zeindeer now matches the zombie deer in its attack speed, the horse and reindeer now also attack slower, 1.3 times a player turn rather than 3 times.

#### Describe alternatives you've considered
Large quadrupeds like the deer and horse could be given their own special attacks, a slower attack like an actual horse kick that does the current damage and also applies a stun/concussed effect. Alternatively we leave their current attack speeds

#### Testing
Current adjustments are somewhat minor, enemies are adhering to their intended attack speeds

#### Additional context
Again, these adjustments are subject to debate, if we want to use a different formula for their attacks or a special attack alternative its entirely within the ability of the JSON to do so. This iteration is significantly scaled back than what I originally intended, however the more complex nature of this balancing leaves it best approached in case by case instances instead of sweeping changes.

### CODE REVIEW

**Comment by PatrikLundell** [comment]
I believe you have to correct the summary. It has to follow the described format as it is parsed and automatically placed into the change log.
"Category" should be replaced with one of the specified keywords (without any quotes) followed by a description (within quotes), except when the keyword is None, when no description should be specified.

**Review by Maleclypse — APPROVED** [review]
This seems reasonable.

**Comment by kevingranade** [comment]
Changes are reasonable, but need to figure out why the checks aren't running.

File: data/json/monsters/mammal.json [hunk 1]
```diff
@@ -1895,6 +1895,7 @@
     "weight": "550 kg",
     "hp": 90,
     "speed": 300,
+    "attack_cost": 225,
```
Comment (Maleclypse) [inline data/json/monsters/mammal.json:1898]:
```suggestion
    "attack_cost": 230,
```
I'm making this slight change to see if that allows the tests to fire.

**Comment by kevingranade** [comment]
This pull request has been mentioned on **Cataclysm: Dark Days Ahead**. There might be relevant details there:

https://discourse.cataclysmdda.org/t/it-took-3-characters-but-i-finally-got-my-first-kill-are-you-proud-of-me/27606/8

**Pull request merged**